superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -595,9 +640,7 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
         dbToUpdate.configuration_method === CONFIGURATION_METHOD.DYNAMIC_FORM, // onShow toast on SQLA Forms
       );
       if (result) {
-        if (onDatabaseAdd) {
-          onDatabaseAdd();
-        }
+        if (onDatabaseAdd) onDatabaseAdd();
```

Comment:
nit, but a little cleaner for next time:
```suggestion
        onDatabaseAdd && onDatabaseAdd();
```